packages/@aws-cdk/aws-iotevents/README.md

```diff
@@ -51,7 +51,7 @@ import * as iotevents from '@aws-cdk/aws-iotevents';
 
 const input = new iotevents.Input(this, 'MyInput', {
   inputName: 'my_input', // optional
-  attributeJsonPaths: ['payload.temperature'],
+  attributeJsonPaths: ['payload.deviceId', 'payload.temperature'],
```

Comment:
Why this change - seems unrelated to the rest of this PR...?

Reply:
I modified this for the sake of reality.
The path passed in `detectorKey` must be declared in this `attributeJsonPaths`. And the value passed in `detectorKey` is often the unique value of the device or device group.
This is because IoT Events is often used to monitor devices and device groups.

I modified integ test for the same reason.